shared/remediations/bash/kernel_module_usb-storage_disabled.sh

```diff
@@ -1,2 +1,3 @@
 # platform = multi_platform_rhel, multi_platform_fedora
+# THIS FILE IS GENERATED. DO NOT EDIT!
 echo "install usb-storage /bin/true" > /etc/modprobe.d/usb-storage.conf
```

Comment:
@redhatrises Thank you for hint, but these issues are not related to this PR (see changed lines).
I have created issue assigned to me for that https://github.com/OpenSCAP/scap-security-guide/issues/1369. 

I am working on some structural changes and I cannot fix such issue during the work.